EIPS/eip-2384.md

```diff
@@ -0,0 +1,40 @@
+---
+eip: 2384
+title: Istanbul/Berlin Difficulty Bomb Delay
+author: Eric Conner (@econoar)
+discussions-to: https://gitter.im/ethereum/AllCoreDevs?at=5dd4ce8050010612b2eeb9d6
+type: Standards Track
+category: Core
+status: Draft
+created: 2019-11-20
+---
+
+## Simple Summary
+The average block times are increasing due to the difficulty bomb (also known as the "_ice age_") and slowly accelerating. This EIP proposes to delay the difficulty bomb for another 4,000,000 blocks (~611 days).
+
+## Abstract
+Starting with `???_FORK_BLKNUM` the client will calculate the difficulty based on a fake block number suggesting the client that the difficulty bomb is adjusting around 9 million blocks later than previously specified with the Homestead fork.
```

Comment:
Why do you say "around 9 million" -- it's exactly 9 million, right?

Reply:
And actually, you might say "9 million blocks later than the Homstead, which is also 5 million blocks later than specified in Constantinople"